deepmd/pt/loss/property.py

```diff
@@ -0,0 +1,151 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import logging
+from typing import (
+    List,
+)
+
+import torch
+import torch.nn.functional as F
+
+from deepmd.pt.loss.loss import (
+    TaskLoss,
+)
+from deepmd.pt.utils import (
+    env,
+)
+from deepmd.utils.data import (
+    DataRequirementItem,
+)
+
+log = logging.getLogger(__name__)
+
+
+class PropertyLoss(TaskLoss):
+    def __init__(
+        self,
+        task_num,
+        loss_func: str = "smooth_mae",
+        metric: list = ["mae"],
+        **kwargs,
+    ):
+        r"""Construct a layer to compute loss on property.
+
+        Parameters
+        ----------
+        task_num : float
+            The learning rate at the start of the training.
+        loss_func : str
+            The loss function, such as "smooth_mae", "mae", "rmse"
+        metric : list
+            The metric such as mae,rmse which will be printed.
+        **kwargs
+            Other keyword arguments.
+        """
+        super().__init__()
+        self.loss_func = loss_func
+        self.metric = metric
+        self.task_num = task_num
+        self.mean = kwargs.get("mean", 0)
+        self.std = kwargs.get("std", 1)
+        self.beta = kwargs.get("beta", 1.00)
+
+    def forward(self, model_pred, label, natoms, learning_rate, mae=False):
+        """Return loss on loss and force.
+
+        Args:
+        - model_pred: Property prediction.
+        - label: Target property.
+        - natoms: Tell atom count.
+
+        Returns
+        -------
+        - loss: Loss to minimize.
```

Comment:
Should use the same format as docs for other methods.

Reply:
I have modified it.